src/test/java/unicash/logic/commands/ClearTransactionsCommandTest.java

```diff
@@ -32,4 +37,40 @@ public void execute_nonEmptyUniCash_success() {
                 ClearTransactionsCommand.MESSAGE_SUCCESS, expectedModel);
     }
 
+    @Test
+    public void multipleClearTransactionsCommand_equalsTrue() {
+        ClearTransactionsCommand clearCommand = new ClearTransactionsCommand();
+        assertEquals(clearCommand, new ClearTransactionsCommand());
+
+    }
+
+    @Test
+    public void sameClearTransactionsCommand_equalsTrue() {
+        ClearTransactionsCommand clearCommand = new ClearTransactionsCommand();
+        assertTrue(clearCommand.equals(clearCommand));
+        assertTrue(clearCommand.equals(new ClearTransactionsCommand()));
+
+    }
+
+    @Test
+    public void equalsMethod_differentCommandTypes_returnsFalse() {
```

Comment:
Minor nit on the unit test name but `equalsMethod` isn't needed since `equals_` is not going to "shadow" the `equals` method and cause problems, we use `toStringMethod` since there's no following details of the test case

Reply:
So `equals_differentCommandTypes_returnsFalse` is perfectly fine!

Reply:
Thanks, I've changed this wherever else I found as well